interpreter/src/test/scala/pl/touk/nussknacker/engine/compile/SubprocessResolverSpec.scala

```diff
@@ -297,6 +295,26 @@ class SubprocessResolverSpec extends AnyFunSuite with Matchers with Inside{
     }
   }
 
+  test("handle subprocess with empty outputs") {
+    val fragment = ScenarioBuilder
+      .fragment("fragment1")
```

Comment:
It looks a little bit different than in scenarios. In scenarios each node is visible as a separate builder invocation, so there is .source. Here .fragment contains this input node inside invocation. Can it be consistent with scenarios or there are some troubles with that?

Reply:
no, it can be done with some code ;) . For fragments the separation just doesn't add any value. I can change it in next PR if you like consistency better

Reply:
It's fine